File: JavaScript/Faccat/Exercicio9.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Digite o salário atual: ", function(salario) {
  rl.question("Digite o percentual de ajuste: ", function(ajuste) {
    ajuste = salario + (ajuste * salario) / 100;

    console.log("O salário após o reajuste é: " + ajuste);

    rl.close();
  });
});

File: Java/src/Faccat/Exercicio9.java
package Faccat;

import java.util.Scanner;

public class Exercicio9 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o salário atual: ");
        double salario = scanner.nextDouble();

        System.out.print("Digite o percentual de ajuste: ");
        double ajuste = scanner.nextDouble();

        ajuste = salario + (ajuste * salario) / 100;

        System.out.println("O salário após o reajuste é: " + ajuste);

        scanner.close();
    }
}